Trunk.toml

```diff
@@ -26,6 +28,8 @@ locked = false
 minify = "never" # can be one of: never, on_release, always
 # Allow disabling sub-resource integrity (SRI)
 no_sri = false
+# An optional cargo profile to use
+# cargo_profile = "release.trunk"
```

Comment:
I don't think periods are allowed in `Cargo.toml` profile names.
```suggestion
# cargo_profile = "release-trunk"
```

Also, you already have an example up above, merge this with that one.